content/en/docs/installation/operator-lifecycle-manager.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Operator Lifecycle Manager"
```

Comment:
note: I'm not at all familiar with OLM, so I haven't reviewed anything technical in this page - just the wording